src/vehicle.cpp

```diff
@@ -3925,91 +3925,6 @@ void vehicle::operate_planter(){
     }
 }
 
-void vehicle::on_move(){
-    if(scoop_on){
-        operate_scoop();
-    }
-    if( planter_on ){
-        operate_planter();
-    }
-    if( plow_on ){
-        operate_plow();
-    }
-    if( reaper_on ){
-        operate_reaper();
-    }
-}
-void vehicle::operate_plow(){
-    for( const int plow_id : all_parts_with_feature( "PLOW" ) ){
-        const tripoint start_plow = global_pos3() + parts[plow_id].precalc[0];
-        if( g->m.has_flag("DIGGABLE", start_plow) ){
-            g->m.ter_set( start_plow, t_dirtmound );
-        } else {
-            const int speed = velocity;
-            const int v_damage = rng( 3, speed );
-            damage( plow_id, v_damage, DT_BASH, false );
-            sounds::sound( start_plow, v_damage, _("Clanggggg!") );
-        }
-    }
-}
-
-void vehicle::operate_reaper(){
```

Comment:
And you deleted the wrong version of those functions, those here are the good ones, pleases delete the other ones.